tasks/setup/vss-extension.json

```diff
@@ -0,0 +1,41 @@
+{
+  "manifestVersion": 1,
+  "id": "setup-infracost-task",
+  "name": "Setup Infracost",
```

Comment:
We are going to have `Infracost Comment` task, maybe we should name this one as `Infracost Setup` to look alike? `Comment Infracost` doesn't look right. What do you think?

Reply:
Yup good spot - just missed this out changing over, but I think `Infracost Setup` is the way to go.

Reply:
done in https://github.com/infracost/infracost-azure-devops/pull/9/commits/95030bf9394c362cb56a6bdec90109b014fcd819